CHANGELOG.md

```diff
@@ -5,6 +5,36 @@ This changelog is according to [Keep a Changelog](http://keepachangelog.com).
 All notable changes to this project will be documented in this file.
 We will follow [Semantic Versioning](http://semver.org/) from version 2 and onwards.
 
+## 1.4: July 19th, 2016
+
+### Changed
+    * Improves feedback texts of assessments.
+    * Improves Russian transliteraion.
+    * Determine length of title based on pixels in stead of on number of characters.
+    * Words comprised of only digits are no longer counted in the Flesch Reading assessment.
+    * Improves passive voice detection by ommiting HTML tags.
+    * Non breaking spaces are replaced with normal spaces
+    * Improves the competing links check in post for linking to the same post.
```

Comment:
I think this should be in the `Fixed` category and should say something like: `Fix a bug in the competing links assessment where a link to the same post would be counted as an outbound link.`